tests/README.md

```diff
@@ -11,5 +11,5 @@
     }
     ```
 * To run the tests contained in `test_users.py` a json with credentials for a user with the role user admin have to be provided in a file named `config_useradmin.json`. The format has to be the same as `config.json`. Both users have to live in the same organisation on the same LANA Process Mining instance.
-* We require the r analyst role to have access to one log named `Incident_management.csv`.
+* We require the r analyst role to have access to one log named `Incident_management.csv`, one shiny dashboard named `incident-test-shiny-dashboard` and one dashboard named `incident-test-dashboard`.
```

Comment:
Has the shiny dashboard to be connected to the log? We should specify this, the same as we can specify that the shiny dashboard does need to have any actual code. We just work with the resource definition/it's metadata.